src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -133,4 +137,20 @@ public static PdaoLoadStatistics getIngestStatistics(FlightContext context) {
     FlightMap workingMap = context.getWorkingMap();
     return workingMap.get(IngestMapKeys.INGEST_STATISTICS, PdaoLoadStatistics.class);
   }
+
+  public static Predicate<FlightContext> noFilesToIngestPredicate() {
```

Comment:
- In general, I wouldn't add the type of something to a name, e.g. `FooClass` or `FooEnum` should be `Foo`. So I would drop `Predicate` from this name
- I would go one step further and make this a static object instead of making a function that returns an object, since the same predicate function (class instance) can be used for all cases

Reply:
I think this is a holdover from development. Its been removed.